pkg/exchange/bitget/types.go

```diff
@@ -425,3 +427,63 @@ type AccountEvent struct {
 	actionType ActionType
 	instId     string
 }
+
+type Trade struct {
+	// Latest filled price
+	FillPrice fixedpoint.Value `json:"fillPrice"`
+	TradeId   types.StrInt64   `json:"tradeId"`
+	// Number of latest filled orders
+	BaseVolume fixedpoint.Value           `json:"baseVolume"`
+	FillTime   types.MillisecondTimestamp `json:"fillTime"`
+	// Transaction fee of the latest transaction, negative value
+	FillFee fixedpoint.Value `json:"fillFee"`
+	// Currency of transaction fee of the latest transaction
+	FillFeeCoin string `json:"fillFeeCoin"`
+	// Direction of liquidity of the latest transaction
+	TradeScope string `json:"tradeScope"`
+}
+
+type Order struct {
+	Trade
```

Comment:
Order struct embeds a Trade struct ?

Reply:
I just want everyone to know that the trade field